### ISSUE TITLE
CLI File with Zero Precipitation

### ISSUE THREAD

**Issue by akrherz** [dailyerosion/dep#307]
Whilst investigating R-Factor for #303, I found three files, so far, with no precipitation.  The files are getting updated daily, but precip is all zeros, le sigh.  These files are all near -87.83E 44.62N, which is here:

![Screenshot from 2024-12-16 12-36-52](https://github.com/user-attachments/assets/34014139-05b1-4469-a668-e56a8b0d4134)

Wonders never cease...

**Comment by akrherz** [comment]
Looking at 2024 IEMRE totals for this point

```
>>> import httpx
>>> import pandas as pd
>>> df = pd.DataFrame(httpx.get("https://mesonet.agron.iastate.edu/iemre/multiday/2024-01-01/2024-12-31/44.63/-87.81/json", timeout=30).json()["data"])
>>> df.sum(numeric_only=True)
mrms_precip_in                34.27
prism_precip_in               30.98
daily_precip_in               23.25
12z_precip_in                 23.34
climate_daily_precip_in       31.58
```

That's sub-optimally far away from PRISM, but still does not explain the zeros.  My assumption at the moment is off-by-one in the stageIV grid reprojection

**Comment by akrherz** [comment]
This looks off-by-one or maybe 1/2 pixel

![prism_pain](https://github.com/user-attachments/assets/e13a8311-6205-453e-a928-0d2dcd0b8f3c)

**Comment by akrherz** [comment]
While I have found a million off-by-one issues, I am now convinced the problem is shown here:

![iemre_trouble](https://github.com/user-attachments/assets/1cdd0fde-c90b-489c-8ec9-0b6bad64a3be)

My IEMRE `hasdata` is too tight.

**Comment by akrherz** [comment]
The horrors never cease...  Now it appears I have discovered pixels within MRMS that are near zero for some years.  For example, 40.31N 98.45W for 2015.

**Comment by akrherz** [comment]
Getting very close to putting a bow on all the various grid navigation fun.
![after](https://github.com/user-attachments/assets/10cd2399-39f9-4918-b1ca-336223732afc)

**Comment by akrherz** [comment]
Summarize current status.

- The near-zero MRMS pixels are limited to just 2015 it appears, and only hits about 4 WEPP CLI Files. They are points over RADAR locations in Nebraska and Kansas.  I wrote a crude script to copy a nearby pixel.
- We had 12 CLI files at domain borders, they were not used, so I just deleted them.
- While I am still unconvinced that I have this all as "pixel perfect", we should be much closer than before.  Famous last words...

So, sticking a fork in this for now.
